source/ccci-stolaf-college/v1/streams.ts

```diff
@@ -94,3 +96,22 @@ export async function archived(ctx: Context) {
 		sort,
 	})
 }
+
+export async function search(ctx: Context) {
+	ctx.cacheControl(ONE_HOUR)
```

Comment:
Do we have an issue with caching with this endpoint? I'm seeing stale results on subsequent searches